src/mapcraftercore/renderer/blocktextures.cpp

```diff
@@ -197,6 +229,22 @@ BlockTextures::BlockTextures()
 	  GLASS_SILVER("glass_silver"),
 	  GLASS_WHITE("glass_white"),
 	  GLASS_YELLOW("glass_yellow"),
+      GLAZED_TERRACOTTA_BLACK("glazed_terracotta_black"),
```

Comment:
I see the indendation here has changed too. Have you added these texture images manually? There is a make target to generate this code if you added the texture images to the `src/data/textures`-directory: Just run `make gen_texture_code` in case you didn't do this.